microsoftgraph/microsoft-graph-explorer-v4
# Fix: Run query loop

## Overview
Fixes #2474 

### Demo
![image](https://user-images.githubusercontent.com/45680252/229801533-6276f4d8-9610-4271-9a01-e20f13188ee5.png)


### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

* How to test this PR
Sign in with a Microsoft account with permission restrictions
Run Query that might return a 401 e.g: https://graph.microsoft.com/v1.0/deviceAppManagement/managedAppPolicies
Notice fix
